ydb/core/tx/datashard/datashard__data_cleanup.cpp

```diff
@@ -0,0 +1,11 @@
+#include "datashard_impl.h"
+
+namespace NKikimr::NDataShard {
+
+void TDataShard::Handle(TEvDataShard::TEvForceDataCleanup::TPtr& ev, const TActorContext& ctx) {
+    // TODO: implement
+    Y_UNUSED(ev);
```

Comment:
Please send `TEvForceDataCleanupResult` from the function.
And add one unit test for it.